app/serializers/external_registry_bike_v3_serializer.rb

```diff
@@ -0,0 +1,63 @@
+class ExternalRegistryBikeV3Serializer < BikeV2Serializer
+  delegate \
+    :external_id,
+    :location_found,
+    :status,
+    :url,
+    to: :object,
+    allow_nil: true
+
+  def stolen
+    object.stolen?
+  end
+
+  def status
+    object.status&.titleize
+  end
+
+  def mnfg_name
+    object.mnfg_name&.titleize
+  end
+
+  def frame_model
+    object.frame_model&.titleize
+  end
+
+  def frame_colors
+    object.frame_colors&.split(/\s*,\s*/)&.map(&:titleize)
+  end
+
+  def registry_name
+    object.external_registry_name
```

Comment:
Naming here makes me think - why not have the same attribute in the database that we use here?

Would it make more sense to just call the attribute `registry_name`?

Reply:
Ditto for `registry_url`

Reply:
Makes sense. Updated in 5933eeb3